.rultor.yml

```diff
@@ -8,7 +8,8 @@ docker:
 merge:
```

Comment:
no benefit in fixing this file.  it needs to be deleted anyway  https://github.com/coala/coala/issues/5328  ; so do that one first